.github/workflows/semantic-pr-title.yml

```
@@ -0,0 +1,30 @@
+name: Semantic PR Title
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)